src/utils/shuffled_queries.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import numpy as np
+import pandas as pd
+from random import shuffle
+
+def run(args_dict):
+	f = open('{}/{}'.format(args_dict['output_dir'], args_dict['file']))
```

Comment:
if you're opening a file here, it's safer as:

```
with open('{}/{}'.format(args_dict['output_dir'], args_dict['file'])) as f:
    clean_txt = f.readlines()
```

as with `with` will ensure a good closure on the file once done reading. Otherwise, you should have a `f.close()` after you `readlines()`.